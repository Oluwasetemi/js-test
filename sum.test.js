const sum = require("./sum");

test("addd 1 + 2 to equal 3", () => {
  expect(sum(1, 2)).toBe(3);
});
test("expect 2 + 2 to equal 4", () => {
  expect(sum(2, 2)).toBe(4);
});
test("two plus two", () => {
  const value = sum(2,2);
  expect(value).toBeGreaterThan(3);
  expect(value).toBeGreaterThanOrEqual(3.5);
  expect(value).toBeLessThan(5);
  expect(value).toBeLessThanOrEqual(4.5);

  // toBe and toEqual are equivalent for numbers
  expect(value).toBe(4);
  expect(value).toEqual(4);
});

test('there is no I in team', () => {
    expect('team').not.toMatch(/I/);
  });
  
  test('but there is a "stop" in Christoph', () => {
    expect('Christoph').toMatch(/stop/);
  });

const shoppingList = [
    'diapers',
    'kleenex',
    'trash bags',
    'paper towels',
    'beer',
  ];
  
  test('the shopping list has beer on it', () => {
    expect(shoppingList).toContain('beer');
  });