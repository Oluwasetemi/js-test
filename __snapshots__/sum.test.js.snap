// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a snapshot of sum(5,3) 1`] = `8`;
